app/components/projects/projects.module.css

```diff
@@ -0,0 +1,11 @@
+.projectsMenu {
+  display: inline-flex;
+}
+
+.projectMenu {
```

Comment:
it would be good to name this or the above a bit differently

they are very similar in terms of name and could lead to some confusion

Reply:
done!